index.js

```diff
@@ -3,17 +3,17 @@ const exec = require('child_process').exec
 const fs = require('fs')
 
 try {
-  installSFDX()
+  installSF()
 } catch (error) {
   core.setFailed(error.message)
 }
 
-function installSFDX(){
-  var download = 'wget https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-x64.tar.xz -q -P /tmp'
-  var createDir = 'mkdir sfdx'
-  var unzip = 'tar xJf /tmp/sfdx-linux-x64.tar.xz -C sfdx --strip-components 1'
-  var install = 'echo "`pwd`/sfdx/bin" >> $GITHUB_PATH'
-  var version = 'sfdx/bin/sfdx --version && sfdx/bin/sfdx plugins --core'
+function installSF(){
+  var download = 'wget https://developer.salesforce.com/media/salesforce-cli/sf/channels/stable/sf-linux-x64.tar.xz -q -P /tmp'
+  var createDir = 'mkdir ~/cli/sf'
```

Comment:
Typically you need the '-p' flag to recursively create directory like this.  Does the "\~/cli" already exist?  Might be safer to add it, just in case in future iterations, the '\~/cli' directory doesn't exist.

Reply:
Yup @stohn777, I see what you are talking about.  I have added an additional step to create the "cli" directory first.